### ISSUE TITLE
User Suggestions

### ISSUE THREAD

**Issue by Speiger** [TinyModularThings/IC2Classic#192]
- [x] Cables Lay fire when burning down because of to much energy went through. Same as Machine Exploding. (With Config Options)
- [x] Crop Harvester. Not easy to automate with just item transport (like hoppers)
- [x] Quarry Scanner
- [x] Pufferfish needs a grin Powder recipe

**Comment by NinjaKap** [comment]
Similar machines to IC2 Experimental, but "Classicified". The latest IC2 Exp. builds have added Tanks, the Advanced Miner (Works like a Quarry), Chunk Loader, Reactor Coolant Injectors, Crop Stuffs, etc. - I think it would be cool to have a few new IC2 Exp. machines, but definitely with changed mechanics and recipes.

**Comment by Speiger** [comment]
@NinjaKap 
* ChunkLoader. No... With that so much could break when wrong done and i do not feel like making it good...
* Classics Default Miner is not so limited like IC2 Exps Miner... I added a lot more Scanners and i could add a Quarry scanner which would make him like the adv miner...
* Reactor Coolant Injectors for these Laps&Redstone cells i know they exists but i think when you want to do these kind of reactors then just move items....
* Whatever you mean with CropStuff...
* I understand what you mean but to point that out i am planning to limit every IC2 Machine LVL (LV-MV-HV) to 15-16 machines. LV is already at 12-13, mv is also close to that and HV has like 5 machines max...

^^" I am trying to make the things that are there much more usuable instead of throwing out much more machines to the game that do the same thing over and over again... (Well AdvMachines is a differend story)

**Comment by NinjaKap** [comment]
Hmm. Actually, now that I think about it, I like your idea wayyyyy better then mine. More usable machines is always fun, I know that I thoroughly enjoy having a bajillion machines on a wall to use. Thank You!

**Comment by Speiger** [comment]
@NinjaKap i have a idea...
Instead of making a fully automated CropFarm i make a Crop Harvester.
What this crop harvester does its letting the crops Drop its item on the ground. And you have to find a way to pick them up. (There is enough ways)
What do you think about that?

**Comment by NinjaKap** [comment]
I like that idea. It would definitely encourage me, and others, to make a creative system. The one request I have for this Crop Harvester is that it outputs to the Bottom, and only the bottom. This way it allows for one single hopper, knowing where it's going, instead of a bajillion hoppers with the items flying everywhere, similar to Better With Mods's Mill Stone. Also, the Crop Sticks allow you to right-click and the seeds remain there, correct? If so, let the Harvester use that system, instead of breaking the crop/cropsticks. if no crop sticks are present, break the crop completely?

**Comment by Speiger** [comment]
@NinjaKap that is a paradox in itself. I am saying that it will drop the drops at the crop & wont pick it up. (So you have to find a way to do it). And you say you like it but you wish to have that they pick it up and push it down (i would rather pick up because its harder to make that astictic good)...

**Comment by NinjaKap** [comment]
Ah, okay. I didn't realize you drop it on the block, not by the machine. I guess the real question is if you want IC2 Classic to be an Independent Mod, that its features can be automated using Vanilla (like Botania)? Or do you want portions of IC2 Classic to depend on other mods to be automated with. In honesty, the harvester has no "benefit" that I immediately see over a simple Vanilla Piston-Water system (Unless using Mods). If this were to be of benefit to the player, while having limitations, I think some Crop Planter (or a variant) needs to exist. Maybe a block in the middle that, when supplied by dropped seeds ON THE TOP ONLY plants in a 1 block radius (3x3, similar to the MFR Planter)? Something that, if only IC2 Classic, can be automated using Vanilla, giving great benefits, but also when used with other mods, can be made much easier and more efficient? Other then that, the Harvester Block would really only compact a build, and requiring Power (another variable to add in to a contraption), whereas a simplistic Vanilla piston contraption wouldn't require any EU, and doesn't require that Power variable, saving the User both Time and Energy. If you don't want a Crop Planter of ANY sort, maybe give the Harvester more benefits OVER a vanilla piston harvester, something like the Harvester does a complete 9x9 area, or larger dependent on Upgrades, where a Piston Harvester can only do 8x(Some Number, dependent on Length)? I guess, to sum it up, if the Harvester has no benefits over something that can be done in Vanilla, it's more likely to be ignored. Sorry for these long walls of text. Thanks for all your hard work and dedication to IC2 Classic. Excited to see the 1.9.4/1.10 Outcome!

**Comment by Speiger** [comment]
> Ah, okay. I didn't realize you drop it on the block, not by the machine. I guess the real question is if you want IC2 Classic to be an Independent Mod, that its features can be automated using Vanilla (like Botania)? 

You know you could run a Minecart track under the crops with a Hopper minecart that pick up all the drops...
Or use other mod methodes to make that happen. Also i am planing on doing something that IC2 Classic could automate that even without Vanilla minecraft...

> I think some Crop Planter (or a variant) needs to exist

Nope. A planter will never exist. Why? Because in IC2 Crops need only to be planted once... So you do that part manually and the harvester takes only the drops without destroying the drops...

> maybe give the Harvester more benefits OVER a vanilla piston harvester, something like the Harvester does a complete 9x9 area,

Idea was  that it starts with a 3x1x3 area (so no height support) & you could expand with upgrades up to 5 times (5x1x5, 7x1x7, 9x1x9, 11x1x11, 13x1x13) Noting the bigger the area the longer it takes the crops to harvest (so it would do up to 4 Crops per second). 

Also i have no problems with long texts. They take longer for me to read & answer but they contain more detail i can pick & answer too....

And now here a compromise: You can put in a Upgrade which allows the Harvester to pick up the items but you have to extract them manually. It will not push them through, and as soon it gets a backlock it will prevent further progress....

**Comment by NinjaKap** [comment]
I'm going to number the following in the order you answered. So, your first answer is 1, second is 2, etc.

1. Yes, I know the trick with the Minecart. I put it in the giant block of text. :D

2. Still a little confused what you mean by this....so, basically, if you plant crops (in cropsticks, I assume, as you said IC2 Crops), you want the Harvester to harvest the crop, but the seed stays as that's how the cropsticks work. If so, that's what I said earlier. "...instead of breaking the crop/cropsticks. if no crop sticks are present, break the crop completely?" That's fine. I like that idea.

3. Well, I can't necessarily answer this one as the Second Answer basically removes the need for this. Since it does have an advantage (only takes the output of the crop, doesn't break the crop itself), this can be thrown out. Don't want to make it too over-powered.

4. Long texts are beautiful, aren't they?

5. Well, if it was miscommunication, then I wouldn't advise implementing that upgrade. Let players be creative, as long as the crop stays there and doesn't need replanting.

I think that our long text blocks were just miscommunication. I am, as you can tell, horrible at wording things as I think, soooo....yeah. Thanks again, Speiger.

Also, I've always loved the IC2 Classic Nuclear functionality. Especially with the new Enriched Uranium stuff, the steam reactor, with the steam turbines, etc - I think it would be fun to have a radiation system similar to Real-Life radiation. 
I know this is VERY difficult, and do not by any means expect you to adopt the idea. But, that being said, I think it would make players actually encase and safeguard their reactors much more. This should also affect the nuke. Basically (Not exact like IRL Radiation, this is my thoughts on how it should be), if a Reactor/Nuke blows up, everybody in a 100-block radius instantly dies. This diminishes with the more protection a reactor has, and the Reinforced Stone can block a little bit of it. Now, in a 200-block radius, all players who haven't died get radiation, the closer they are the worse it is. Hunger depletion, health going down, etc. - this should affect all animals aswell. 

After the initial blast, the area should be contaminated. The contaminated area is diminished with the protection of the reactor. You MUST enter the contaminated area with a Hazmat Suit or alternative. The durability/power starts running down. They have until the durability/power runs out to retrieve the valuables from the zone, before they get radiation. That then makes the area unlivable, forcing the players to move out. 

If you wanted, as an extra challenge, is to include "half lives." The Radiation (maybe over an hour?) cuts itself in half. So, a contaminated area of 200-blocks radius would shrink to only 100-block radius, and so forth. These numbers should be tweaked. After (for our example, 25-blocks), the radiation disappears and the area is livable again.

Please, do yourself a favor and DO NOT add this to the initial 1.9.4 Update. If you do (for whatever reason) like the general basis of the idea (as I know I'm horrible at explaining things), add little bits and pieces over time. It'd be one hell of a feature, but would make reactors (and nukes) a little more....dangerous. As I said earlier, THANK YOU for reading this horribly long text thingie and keep up the good work! Now to try and trim this thing...

Sorry for this. Adding this 20 mins after the above thing. If you're not using MCMultiPart, please do! Chisels and Bits & IC2 Cables would be fun to do. Sorry and thanks!

**Comment by Speiger** [comment]
> Still a little confused what you mean by this....so, basically, if you plant crops (in cropsticks, I assume, as you said IC2 Crops), you want the Harvester to harvest the crop, but the seed stays as that's how the cropsticks work. If so, that's what I said earlier. "...instead of breaking the crop/cropsticks. if no crop sticks are present, break the crop completely?" That's fine. I like that idea.

It will be very basic. It will ignore everything that is not a cropStick. And if it is a crop stick it will check if it can do the harvest. If it can it does. If not then it ignore it...
And the harvest will reset the crop to the state that the crop says. (Simply simulating a rightclick)

> Long texts are beautiful, aren't they?
Depends...

> Also, I've always loved the IC2 Classic Nuclear functionality. Especially with the new Enriched Uranium stuff, the steam reactor, with the steam turbines, etc - I think it would be fun to have a radiation system similar to Real-Life radiation. 

Now this includes also the stuff that is bound to this text to (but it would be to big to copy):
I am not going to implement that. Main Reason: Yeah Radiation should be a thing but it should not annoy people. Reactor have already the Complex Stomp even if they are not really complext. For people who are new to this mod its clearly a "I am not going to use this feature case" and i personally hated the constant radiation effect too... If you want that: Thats whats Exp Is for. You want 100% Realism there you can have it.
Also i think the damage that a Nuclear Reactor does is already enough. And also keeping chunks (that would be what i have to do) radioactive seem maybe cool but also takes in a lot of drag because you constantly have to search for entities and all that stuff. I know it can be done efficent but its not as easy as you think... 
Simply: That is not going to happen... (Design Choice)

To your multipart request... I am not thinking that this will be a feature at any time... Not that multipart is a bad idea but first: IC2 Classic will have no ModRequirements & As Cool multiparts are i always had issues & bugs etc with it and i do not want to get into issues with IC2C just because their mod is not stable (Ignoring the stable lvl of it). Also another thing: IC2C Networking system which i rely on is not compatible with any kind of multipart system (by default)... Its requireing the TileEntities itself... Not a fake TileEntity in a TileEntity... 
So i think this is not going to happen....

**Comment by NinjaKap** [comment]
kk. The harvest idea seems like a cool idea.

Yeah, I figured you wouldn't add it, but I had to ask :p - I snipped a LOT of that, to shorten it, but some alternatives were to have it in the Config as Disabled by Default and other things - don't worry about it, just a suggestion. I'm completely fine if it doesn't get added. The Nuclear Physics are quite cool by themselves :D 

And McMultipart may be getting into Minecraft Forge shortly, unless that's been changed and I haven't heard about it. And I hadn't thought about it as a Fake TileEntity in a TileEntity....hmpf.....

Thank You SOO much for all your work on IC2 Classic. I'm very excited to play around with the new update.

Quick Question: When I played in 1.7.10 with 1.2.0.2, I noticed the Bronze Cabling was really just a better version of the Copper Cable. It was the last map I had been on with 1.7.10, and figured to check out ALL the features I hadn't played with. Are you planning on implementing machines that REQUIRE the bronze cabling? Even though Copper Cables may suffice with the EU I/O, are you planning on giving players a better reason to use the bronze cabling except just less loss per block (and crafting recipes)? Or am I just stupid and haven't realized they do have another use?

**Comment by Speiger** [comment]
@NinjaKap 
1. No Problem.
2. ok. Good to know. By the way a condition/rule i have is: As long it does not change the main feature it can be implemented. Thats why the new Uranium Rods were possible because they were a optional buildup.
3. I know that but even then i am not sure... It is more likely to happen when it is intigrated into forge but even then its not said for sure... Because forge does still so much things not good or has issues which let me think: No thanks... And yeah. Fake TileEntities are something differend then MCMultipart uses but the idea is the same. Just that FakeTileEntities are way more dangerouse the MCMultiparts way because of the new Capabilties system but they are more compat then MCMultiparts...

4. Well when i am done with the 1.9 update we make a testserver and i need a couple people to help me test...

5. Hehehehe you missunderstood the Bronze Cable. Its actually worse then Copper cable. Simply its a weaker but cheaper Gold Cable allowing you if you really want to to step up to MV (128EU) but with higher loss then gold cable (goldCable has a higer loss then CopperCable) so yeah... And i am trying to intigrate it also to recipes... But not sure what i have to think of it....

**Comment by NinjaKap** [comment]
:D 

That's an intelligent philosophy. That way, people who don't want change don't get change, but people who want more features can get more features.

Yeah, even tho McMultipart may be glitchy and crap, it's still better then Forge Multipart. That was a nightmare. Even for players.

If you need people for a test server, I'm open.

So a Bronze Cable is just a weaker and cheaper Gold Cable. Hmpf. WAILA Plugins LIED to me!didn

Thank You! Excited to see what you have in store for us! :D 

*Ignore this is you Wish. It's another Suggestion. O.O* You said in an Earlier post that you were interested in adding different machines for the different Voltages, instead of better versions of the machines for the different Voltages. Would something like an I.D.S.U/Tesseract be possible? I've always loved the Tesseracts from Thermal Expansion, and the Dimensional Transceiver from EnderIO, and when TechReborn became Semi-Stable on 1.7.10, the I.D.S.U I saw (from the origianl GT) intrigued me. Unfortunately it was broken. But, would something like that be possible as an HV+ Tier machine? Send power in, sends it out on the other side. Could work like a Teleporter, where you need to supply it a redstone signal, Storage Unit on the side of it, and use a Freq. Trans to "bind" them together in the world? Maybe? If not completely fine; I know that I could just use Mekanisms (As it's now out for 1.9.4 and up :D) Universal Cable into EnderIOs DImension Transciever, then back to EU, but if there was a variant in IC2C native, that'd be cool. Just a Suggestion, if you don't have any ideas for the Higher Tier Machines. Anyways, thanks again!

**Comment by Speiger** [comment]
I will skip everything except the suggestion because there is no need to answer them...

First you do not need another mod to transform from EU To RF & Back...
Another thing, i had already ideas how to do it but i never really liked it or had not the time to do it.
Here is the thing: Every machine that i add or something delays the release because after the release i want to decrease the work time i put into IC2C drasticly. Simply because i have things to do but also i have a other couple projects (nothing with minecraft) that i want to work on and IC2C will proberly get only the minium required time to support it & keep it running (BugFixes, small version updates everything that can be done in 1 day of work) So yeah...
(Just to point that out i am working on the last 2 updates since last year (i had 2 weeks pause between the first update and this update) and that most of the time dayly)

**Comment by GeneralAtrox** [comment]
**Quality of Life**


The main problem I've always had with IC2 is it's readability with power and processing displays. Whacking in a pile of Overclocker upgrades always provided you with a huge EU power % rather than a flat number. I found even when I tried to work out the power value myself, It never seemed to be the amount required for the machine. Also I was never sure how the transformer upgrades worked. When using IC2 during Skyblock, my method of working out what machine takes what involved placing an upgrade in 1 at a time till the power storage started depleting. 


So is it possible to get these displays updated to represent their specified machine?

**Comment by Speiger** [comment]
@GeneralAtrox have you ever played with IC2 Classic?
I can give you the answer. It is no. Because then you would not tell me about these things. IC2C has his own Waila Plugin which tells you to have a EU reader to be equppied & it tells you the exact numbers, perfectly synced so that you always know how much you need...
Please play with IC2Classic before you ask for stuff that was already implemented.

**Comment by ForumWarrior** [comment]
cross mod compatibility:
- thaumcraft aspects
- ticon tool electric modifier

vein-mining firing mode for Mining Laser

higher tiers of Electric Wrench so I don't have to recharge the damn thing so much

would be nice to have full suit of composit armor instead of just a vest

optional friend access for personal safe

**Comment by NinjaKap** [comment]
Speiger, sorry if you didn't want me to answer ForumWarrior, but I nearly died when I saw it.

ForumWarrior, please read below:

1. ThaumCraft isn't even OUT for 1.9.4, and Azanor is stressed on time. I also don't see what IC2C would be able to do with it. If there's something I'm missing, I'm sorry, but Thaumcraft is a MAGIC mod.

2. Tinker's Electric Modifier worked in 1.7.10. Whether he's implemented it for 1.9.4, I don't know, I don't see the code, but I'm sure Speiger will. He did say "All 1.7.10 features" (unless it's impossible).

3. Vein Mining firing mode? Use Vein Miner! It's an independent mod. It exists! Also I don't know that lasers could bounce off blocks....

4. *soarynPalm* There are BATPACKS and LAPPACKS!

5. Hmm, full composite armor suit....This would be a Speiger answer....

6. There's this helpful mod....called FTB Utilities....that includes chunk claiming for friends access.....I mean, in Native IC2C this might be nice, but why?

Sorry again Speiger, and delete this post if you wish.

**Comment by ForumWarrior** [comment]
NinjaKap

From the wiki tab: "IC2 Classic is a Mod that is rebuilded by the Mod IC2 from Minecraft 1.4.7 and Updated to the version you playing. (1.6.4, 1.7.10)"

I was assuming this was for 1.7.10.  Am I wrong?  If not why are you talking about 1.9.4.

Also Thermal Expansion and Mekanism and EnderIO exist and those are other mods too so lets just throw out energy generation, ore doubling, automated mining, armor, weapons, and anything else another mod already covers.  Do you see why what you suggested is stupid?

**Comment by Speiger** [comment]
@NinjaKap hehe thanks for taking over a part of the answer i will answer on your and @ForumWarrior post.
But first:
@ForumWarrior it seems that you have actually not played much or not at all with IC2 Latest version because there is a full composite suit already implemented...

Now to your suggestions:
* 1: Even so the API is not updated to 1.9.4 i still can use the 1.8.9 API for prefabing it before Thaumcraft comes... But i actually wait on a feature of thaumcraft to make it easy on me. (Because thaumcraft uses a ProjectE like system to access items in a recipe and the Providers.) If that is a thing we can talk about it.
* 2: Forgot to add that to the todoList. Thanks.
* 3: Never used Vein Miner never plan to use it and since i have no idea how that should work nope...
* 4: There is a tier 3 Wrench planned but that will have no Lossless mode at all. It has a way higher chance modifier then both kinds... But try to find a recipe that makes it not early Game. (UU-Matter can be gotten real easy Early game)
* 5: Skipped
* 6: IC2C does not plan to have a friend system. I could make that people can open your save and look at it but take nothing out but that would be not friend based. Then you should then use other mods in combo.

> I was assuming this was for 1.7.10. Am I wrong? If not why are you talking about 1.9.4.

Please look at the milestone that this post is bound to and read its description...

Anyway it would be nice if you play with IC2C before you actually ask for features that are actually implemented. Happend now twice on this post...

**Comment by Reoseah** [comment]
Small idea I had a long time ago. 

Make electric tools combined from engine/battery and its working part. E.g. there is line of hands like this:
 - Common (like all ic2 tools have)
 - Advanced (maybe using carbon plates and energy crystal)
 - Quantum (using iridium plates and lapotron crystal)
 - maybe other, for example I had an idea of small energy condensators that can be quickly replaced, inspired by Half-Life 2 xD

And for working part can be something like this:
 - Bronze one
 - Iron / Steel one
 - Diamond covered one
 - Iridium one (or quantum/laser thing like Laser Drill in Terraria 1.3 update, or something magnetic like Wajra in - if I still remember - Gravitation Suit; or even Sound Screwdriver for wrench/treetap... and I'm not talking about zero-level manipulation of gravitational field! xD)

Maybe some updates can be used, which could be more like Tinker Construct with industrial flavour, or some working parts with special trait that could require more energy, e.g. higher fortune or... hitting zombies with electricity :3

I someday tried to make some textures, maybe they could be useful. They are very simple however.
![engine_quantum](https://cloud.githubusercontent.com/assets/20323084/17374262/74085d1c-59b4-11e6-9a0a-46c6dfbcd26d.png)
![engine_simple](https://cloud.githubusercontent.com/assets/20323084/17374264/74091a2c-59b4-11e6-9f5b-3ea5007ddf7e.png)
![engine_nano](https://cloud.githubusercontent.com/assets/20323084/17374263/7409325a-59b4-11e6-8a54-585db662eca1.png)
![coil_copper](https://cloud.githubusercontent.com/assets/20323084/17374265/74159356-59b4-11e6-97fb-101780381413.png)
![coil_superconductor](https://cloud.githubusercontent.com/assets/20323084/17374261/740851f0-59b4-11e6-8348-d93de3c60e4e.png)

**Comment by Speiger** [comment]
@Reoseah i did not understand your suggestion. Could you get into more detail? What do you exactly mean?

**Comment by Reoseah** [comment]
You craft any drill hand. Then you craft of drill head. Then you combine them somehow like forging tools in Tinkers Construct. 
Hand gives eU storage, head gives mining speed. 
Everything else I wrote was just different ideas about it.

**Comment by NinjaKap** [comment]
@Reoseah That's literally what the mod Super Multi-Drills does. It uses RF, but has the same concept. Just use some Power Converter of some type, into a Charging Station of some type, put the drill in and works how you want.

**Comment by Reoseah** [comment]
@NinjaKap I haven't seen it. And it actually quite far from idea I had. And it is seems to be too cheaty ;D

**Comment by Speiger** [comment]
@Reoseah so you request a ModularDrill addition?

**Comment by NinjaKap** [comment]
> You craft any drill hand. Then you craft of drill head. Then you combine them somehow like forging tools in Tinkers Construct. 
Hand gives eU storage, head gives mining speed. 
Everything else I wrote was just different ideas about it.

That is EXACTLY what Super Multi-Drills does. You gather a Drill Head, which gets the Mining Speed, there's a Motor, Color thingie, and a battery. I love the idea, don't get me wrong, but that negates my love for the tools. If it were "upgrades" to it, that'd be fine. I don't like the idea of more crafting for one thing. That could be an Exp. feature :p

**Comment by Reoseah** [comment]
@Speiger Yep. But not only drills - also chainsaws, electrical wrenches and maybe tree taps. Instead of fixed recipes for electrical tools. At least customizing what battery is used.

**Comment by Speiger** [comment]
@Reoseah well the problem here is that if i am going to implement it it removes the tier system ic2 has. So you simply skip the first & second tier and go directly  to the modular last tier.
Dont get me wrong the idea is interesting but has some flaws...

**Comment by Speiger** [comment]
Also why dont we move this talk to the IRC channel? So we can discuss this faster :3. (If you want to, because i assume here will join a lot more suggestions sooner or later)

**Comment by NinjaKap** [comment]
There's an IRC Channel?

**Comment by Speiger** [comment]
IC2-Classic. Its also listed in the ReadMe...

**Comment by NinjaKap** [comment]
hmpf. Shows you my intelligence level

**Comment by Speiger** [comment]
this has nothing to do with Intelligence... If you compare my typing level then i should be called stupid

**Comment by NinjaKap** [comment]
:p was a joke. Sorry!

**Comment by Reoseah** [comment]
Small suggestion about nether stars and reactors. I've seen there are some enriched nether star uranium in lang files... have no idea how that is working...

Add new reactor part. It is crafted from nether star (or from something else, if it will work here well). It can be just as fuel rods, but white inside, or it can be unique part.
It produces neither energy nor heat. But it causes nearby uranium cells to tick twice (as if two uranium cells were placed near third). By doing this, the part will allow to increase effectiveness of reactor by the cost of nether stars. (Somewhat similar to Neutron Reflector, but emitting instead of reflecting...)

The narrative there is simple: nether star emanates energy/radiation that causes reaction in uranium.

Then I suggest two other things: 
First, reuse spent parts to get some resource. E.g. spent nether star rod - extractor -> some crystals... Then use it to create some endgame items. 
If it will be the main way to craft the items, it grants great tool to make them not too cheap. And on the other side, it will force player to create special kind of reactor for to obtain it. 
Second, add end-game crafted from iridium and other materials version of this part. It can use some energy of reactor, in other words consuming energy instead of producing it.

**Comment by Reoseah** [comment]
One crystal or dust from one rod, using one rod for about 3 irl hours, one-two crystal for one item.

**Comment by Reoseah** [comment]
Also: some pretty hard to select plant can produce this material in form of nuggets or tiny piles

**Comment by Speiger** [comment]
> Small suggestion about nether stars and reactors. I've seen there are some enriched nether star uranium in lang files... have no idea how that is working...

Well its a very dangerouse Fuel Rod. It produces a ton of power but it also produces a ton of heat...
I mean really much heat that you almost can not substain a 2x quad rod without the RSH cells...
So yeah thats how they work.

> Add new reactor part. It is crafted from nether star (or from something else, if it will work here well). It can be just as fuel rods, but white inside, or it can be unique part.
It produces neither energy nor heat. But it causes nearby uranium cells to tick twice (as if two uranium cells were placed near third). By doing this, the part will allow to increase effectiveness of reactor by the cost of nether stars. (Somewhat similar to Neutron Reflector, but emitting instead of reflecting...)

I know what you mean. But i am unsure about this: 
- First it would defeat the point of Reflectors. Because a double tick is better then 1 Reflection boost.
- Second: Heat production will go nuts since it double the heat production per heat & using more then these will let it go nuts

> Then I suggest two other things:
First, those reuse spent parts to get some resource. E.g. spent nether star rod - extractor -> some crystals... Then use it to create some endgame items.
If it will be the main way to craft the items, it grants great tool to make them not too cheap. And on the other side, it will force player to create special kind of reactor for obtaining it.
Second, add end-game crafted from iridium and other materials version of this part. It can use some energy of reactor, in other words consuming energy instead of producing it.

This idea sounds interesting. And it would solve some recipe issues i am having with some items i want to add...
But the problem is: This feature is a optional feature... It would not be relyable and force people to use that new system...
If there would be another way (like plants you suggested) it could be more relyable xD

**Comment by Reoseah** [comment]
I assumed hat nether stars are quite hard to obtain, just as they are in vanilla minecraft. 

Similar but slightly changed idea:
Fill prismarine into fuel rods. Radiate them in reactor for some time. Put in extractor to get new resource. Also alternative ways to obtain it from plants or other things.

Have you know: emeralds change color to deep-blue by irradiating them with neutrons/gamma-rays, beryls of such color known as maxixe or augustite.

**Comment by Speiger** [comment]
Well thats a new thing...
Here is the problem: What do we get out. It should not be to extreme... Also how fast should it be accessable xD

**Comment by Reoseah** [comment]
So I think it can be bright-white or light-blue crystals. I had different textures of this kind unused, so I could search for them in old files... I would suggest names like "Crystallite" or some real world names like "Cryolite".

 If it is not very expensive and end-game, I would suggest some "sink" for it, e.g. good wire, perhaps even superconductor-like.

**Comment by Reoseah** [comment]
Also sinks are often overlooked concept for end-game things. 

Imagine that you have began game focusing on IC2, you've made large energy production, created mass fabricator, made quantum armor... And then you go explore other mods. But iridium production is still working! And what can you do with iridium, when you already have everything from it?

**Comment by Speiger** [comment]
True that is a point. And i am trying to find things which can expand the end game.
Maybe a item that you have to combine with Iridium and maybe silver to get a new type of material that is showing a tier 3+? (Not tier 4 i mean tier 3 but requireing extra stuff)

**Comment by Reoseah** [comment]
So, in files on PC I found this and I have no use. Also strange sphere as bonus xD
It is just recolored flint texture.
![blue_flint](https://cloud.githubusercontent.com/assets/20323084/18198926/cbb9f002-7106-11e6-855d-f1f64f6fb349.png)

**Comment by Reoseah** [comment]
It is now very bright as I would do for things from nether star, but may work well for something from prismarine.

Also, it may generate in The End in small amounts.

**Comment by Speiger** [comment]
Hmmm that would be something xD
Well classic is for this week not under development xD so waiting time now xD

**Comment by Reoseah** [comment]
Small slightly strange suggestion.

Make new fishes that can be processed to get some resources, e.g. metals, grin powder, prismarine and glowstone. It can be viewed as alternative for IC2 plants. 

They can be named in "naive" style like "Silverfish", "Copperscale" and "Ironthorn" or something latin as crops of IC2 are named.

Grin Powder can be created from vanilla Pufferfish, that has some sense as pufferfish contains lethal poison.. (it isn't poisonous for plants, but that can be ignored...)

For metal-producing fishes, I would suggest make them catchable only in deep ocean, like 30-40 blocks   deepness, the deepest place in oceans. (IRL, metals are often concentrated at the bottom of ocean and conglomerations on the shallow places in ocean are an important source of some metals, like Manganese; but the deeps are usually not profitable due to expensiveness of such process. Also there are many species, that live at 500-2500 meters deep underwater, usually near underwater volcanoes, and some of them are really strange.).

You can even create iridium fish! Process in into tiny dust pile of iridium, 4 fish = 1 iridium...

**Comment by Reoseah** [comment]
Also found this two icons which I haven't planned to use anywhere.
![cr_2](https://cloud.githubusercontent.com/assets/20323084/18224386/20ff239a-71dc-11e6-9155-212206236df0.png)

**Comment by Speiger** [comment]
@Reoseah i like some parts of the idea. 
But i have to tell you 1 thing: IC2 Classic is not based on 100% realism... 
You are suggesting a lot of things that are highly Realsistic xD
Not that i do not like some of these ideas but you should not go 100% overrealistic... (Thats what IC2Exp went with)

To your suggestions:
>Make new fishes that can be processed to get some resources, e.g. metals, grin powder, prismarine and glowstone
Grin Powder can be created from vanilla Pufferfish, that has some sense as pufferfish contains lethal poison.. (it isn't poisonous for plants, but that can be ignored...)

Well i agree on that one that the pufferfish should have a recipe for grinpowder but not more extreme then that...

>For metal-producing fishes, I would suggest make them catchable only in deep ocean, like 30-40 blocks deepness, the deepest place in oceans. (IRL, metals are often concentrated at the bottom of ocean and conglomerations on the shallow places in ocean are an important source of some metals, like Manganese; but the deeps are usually not profitable due to expensiveness of such process. Also there are many species, that live at 500-2500 meters deep underwater, usually near underwater volcanoes, and some of them are really strange.).

I think that is not even possible since you only have world & luck access. No position stuff with the Standart Loot table stuff...

About all the textures:
They look awesome. I would have some use and ideas what you could do with these. Problem is i try to keep the 16x16 textures so weak computers can use it too...
So it would be great if you can shrink these textures down because 128x128 is way to big...

As soon i get into ic2classic coding again i am going to try to explain what i am going to do ^^"
But i make some teasing markers

**Comment by Reoseah** [comment]
I've scaled them only to be not too small in posts, so nobody will need to open them or scale page up in browser. ) 
Not best of my works, but the other ones I like I'm going to use in my magic mod or in future for some ideas.. xD

They are all 16x16 in original of course. Also I have some of them in different colors  and variations 
![ceruclase_shard_bottom](https://cloud.githubusercontent.com/assets/20323084/18224809/9fad8098-71ea-11e6-9fb4-4915def29a44.png)
![cryolite](https://cloud.githubusercontent.com/assets/20323084/18224810/9fada64a-71ea-11e6-9250-571d2a358537.png)
![ceruclase_shard_top](https://cloud.githubusercontent.com/assets/20323084/18224808/9fad5c80-71ea-11e6-8ad1-8ed381038463.png)
![virtutium_shard](https://cloud.githubusercontent.com/assets/20323084/18224811/9fadbfcc-71ea-11e6-8f42-9a3d8a170c8f.png)

**Comment by Reoseah** [comment]
I've thought there could be event or some more general even on clicking with item, that can be used to override minecraft fishing code

**Comment by Reoseah** [comment]
![20ff239a-71dc-11e6-9155-212206236df0](https://cloud.githubusercontent.com/assets/20323084/18224863/b3faf04c-71ec-11e6-8d13-cc704149f64e.png)
16x16

**Comment by Reoseah** [comment]
![flawless_quartz](https://cloud.githubusercontent.com/assets/20323084/18224903/4ac93262-71ee-11e6-933a-9d44061582f8.png)
2 simple things in same color

**Comment by Speiger** [comment]
User suggestions closed. No new things accepted.
